### ISSUE TITLE
Create the Jakarta EE 9 release page (redesign of Jakarta EE 8 release page)

### ISSUE THREAD

**Issue by shabnammayel** [jakartaee/jakarta.ee#920]
We are revamping the Jakarta EE release page for the release of Jakarta EE 9 on Dec 8. 

Here is the detailed timeline for the Jakarta EE 9 release page: 
https://docs.google.com/spreadsheets/d/1ZPHQLmjdQolzSUJk78lDJpMAyK-T917S25ErQdMaqRc/edit#gid=138610214

Here is the link to the approved mock-up: 

https://projects.invisionapp.com/share/DTZ0N31P6A2#/screens/435142525

Here is the link to the source files to download: https://www.dropbox.com/sh/lt345w4vgp5w7m8/AAAWqHC37lgub4Jie8Za4X_Ya?dl=0
• The design file is called: Jakarta-Design-FINAL.ai

Please note that as discussed with Chris for the "Community Powered Innovation" section, we would like to feature the images of Jakarta EE committers randomly (6 or 12 images at a time and every time that the page reloads it features another set on a random basis). 

We are also looking into animating the homepage banner and adding video bookends (a custom intro and outro including animating the logo) to this page. I will update the issue once I get confirmation from Chris that this is doable.

**Comment by flora8984461** [comment]
Hello @shabnammayel 

I wonder if you can create the image that combines the image in the following screenshot and half logo with overlapping effect? 
![image](https://user-images.githubusercontent.com/39588094/97044841-b5612700-1542-11eb-9883-bcea23eff732.png)

I have built the effect with code like in the preview: https://deploy-preview-926--jakartaee.netlify.app
But I think from pixels and responsive view, it would be great if you can create a combined one for me. 

By the way, do we really want to change the footer? @chrisguindon Since normally footer stays the same. Just hope to confirm.

Thanks!

**Comment by flora8984461** [comment]
@shabnammayel Could you please also provide the icons in your mockup? I did not find them in the dropbox.
Not sure if you can invite me (email: yi.liu@eclipse-foundation.org) to this invision work? I probably can inspect the colors, pixels in detail, or get icons directly through invision.

Thanks!

**Comment by flora8984461** [comment]
@shabnammayel  Thank you for your help and I get the answer. The issue is that I have not paid for Illustrator so that I did not open the .ai file to get the images.

But I do wonder if we have a paid license activation code to use illustrator? @chrisguindon

**Comment by chrisguindon** [comment]
@flora8984461 Yes let's discuss the Illustrator license issue on Monday! Please remind me as I might forget!

**Comment by flora8984461** [comment]
@shabnammayel After our discussion, we wonder if it possible to provide a .psd file as well? Thx!

**Comment by shabnammayel** [comment]
.psd file for which one?

**Comment by flora8984461** [comment]
> .psd file for which one?

For the whole layout. Like the pdf and ai file

**Comment by chrisguindon** [comment]
@shabnammayel Can you also confirm that all the assets such as images and fonts are open source?
https://www.dropbox.com/sh/lt345w4vgp5w7m8/AABFetUpLDh3NMMTmB4Webwba/Fonts/Vitesse?dl=0&subfolder_nav_tracking=1

I am just want to make sure that we have the proper permissions to distribute these assets in our git repo before we publish the site.

**Comment by flora8984461** [comment]
@shabnammayel I saw the email from Victoria, and we think we are good to just use ai files. Thank you so much for your help.

**Comment by chrisguindon** [comment]
@flora8984461 @shabnammayel What has been decided in regards to the PSD?

**Comment by flora8984461** [comment]
> @flora8984461 @shabnammayel What has been decided in regards to the PSD?

Eric can help me with the imgs I need. So that I don't really need a PSD then. Since the designer says creating PSD needs new work.

**Comment by chrisguindon** [comment]
> > @flora8984461 @shabnammayel What has been decided in regards to the PSD?
> 
> Eric can help me with the imgs I need. So that I don't really need a PSD then. Since the designer says creating PSD needs new work.

+1 @shabnammayel let's take note of this and make sure that we ask for a .PSD file to be delivered next time.

**Comment by flora8984461** [comment]
> > > @flora8984461 @shabnammayel What has been decided in regards to the PSD?
> > 
> > 
> > Eric can help me with the imgs I need. So that I don't really need a PSD then. Since the designer says creating PSD needs new work.
> 
> +1
> 
> @shabnammayel let's take note of this and make sure that we ask for a .PSD file to be delivered next time instead of an illustrator file.

I'm not 100% sure if gimp opens .PSD file very well, probably before that, let me have try to download some free psd files to have a try with gimp

**Comment by chrisguindon** [comment]
@flora8984461 We've always gotten .psd for design files! If Gimp does not work, we will get you a copy of Photoshop but let's cross than bridge once we get there!

**Comment by autumnfound** [comment]
> Hello @shabnammayel
> 
> I wonder if you can create the image that combines the image in the following screenshot and half logo with overlapping effect?
> ![image](https://user-images.githubusercontent.com/39588094/97044841-b5612700-1542-11eb-9883-bcea23eff732.png)
> 
> I have built the effect with code like in the preview: https://deploy-preview-926--jakartaee.netlify.app
> But I think from pixels and responsive view, it would be great if you can create a combined one for me.
> 
> By the way, do we really want to change the footer? @chrisguindon Since normally footer stays the same. Just hope to confirm.
> 
> Thanks!

@shabnammayel looking at this design, the visual accessibility is pretty low and we should look at ways of improving it. The offender, in this case, are the white elements that are present on the boat overlay. Here is an accessibility check of the [dark colour](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=F98F1A) and the [light colour](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=FCBF53) of the boat logo in comparison to the white text.

While this could be made very slightly better with a different level of opacity and some strategic darkening, the real problem for this accessibility issue (and most of our other ones globally) is the white text. Using even the [dark blue](https://webaim.org/resources/contrastchecker/?fcolor=171C77&bcolor=FCBF53) will give us fantastic and accessible contrast. WDYT?

**Comment by autumnfound** [comment]
Also, additional context, for WCAG AA compliance (which is the type we aim for), we want a 4:1 ratio at least. Right now we don't crack even 2.5 in best case scenario.

**Comment by shabnammayel** [comment]
This looks okay to me, https://deploy-preview-929--jakartaee.netlify.app/release/. Do we need to change the font to dark blue?

**Comment by autumnfound** [comment]
To be accessible? Absolutely. It's that or we change the design in a way that the background is darker near the text (which would break our colour branding guidelines likely). As we are still under 50 people, we don't have to be accessible by law yet, but the keyword here is yet.

**Comment by flora8984461** [comment]
this is a separate image and background: https://deploy-preview-926--jakartaee.netlify.app/release/
this is a combined background: https://deploy-preview-929--jakartaee.netlify.app/release/

**Comment by trulymobilenwp** [comment]
Not seeing any contributors listed on either of the above previews. Should I?

**Comment by shabnammayel** [comment]
@trulymobilenwp There is a separate issue for the contributors list I think https://github.com/jakartaee/jakarta.ee/pull/923 
There is still an ongoing discussion there

**Comment by flora8984461** [comment]
@shabnammayel @trulymobilenwp 
Yes. After #923 is done, I will migrate them here. 

Thanks!

**Comment by flora8984461** [comment]
Hi @shabnammayel @chrisguindon ,
Yesterday after meeting, if I remember correctly, we want to remove `LINES OF CODE`? Just hope to confirm, thanks!

![image](https://user-images.githubusercontent.com/39588094/97463786-7eaa5880-1916-11eb-9688-054fa1fbfdd8.png)

**Comment by aeiras** [comment]
Dias to everyone working on this ticket :) 

Where is the data coming from? 

The stats should focus on the chosen tracking mechanism that the Spec Committee chose when implementing the release's work, correct? 

**If so:** 

- Jakarta EE has at least [35 Specs ](https://github.com/orgs/eclipse-ee4j/projects/17#column-7346779)that are part of the 9 release. 

- Jakarta EE lines of code are incorrect

- Jakarta EE git repo is incorrect

Something that has become the core of the Contributors' Card PR is that this page allows us to start the separation of Glassfish from Jakarta EE. It matters that the stats are protected & focused solely on the active Specs that are a part of the 9 release. 

A reminder that Jakarta EE is a project, a standalone project under the EE4J.  EE4J is the parent umbrella that ought not to be taken the highlight under the 9 release work. 

Ideally, this and future Jakarta EE releases help us clean up the message via stats representing the project as StandAlone & further finally establish a clear **separation and distinction** to understand better and maintain the project. 

Thanks for the follow-up,

**Comment by chrisguindon** [comment]
Holla @aeiras :)

> Where is the data coming from?

Our focus right now is to implement the design that was approved as-is:
https://projects.invisionapp.com/share/DTZ0N31P6A2#/screens

I don't believe that this content is final. I expect that we will receive requests to edit the content before we go live with this new page.

@shabnammayel / @karenmcnaughton Are the stats provided in the mockups accurate? 

> The stats should focus on the chosen tracking mechanism that the Spec Committee chose when implementing the release's work, correct?

I cannot speak to this since we were not involved in the content creation process. I defer to @shabnammayel / @karenmcnaughton!

**Comment by aeiras** [comment]
I fully understand you Chris!

My questions are directed to the ones providing the data :) You and your team are so amazing at collaborating to implement new page. Thank you for always being fun and quick and caring to ask away and enable this and similarly follow ups. I deeply adore working with you and your team Chris!

**Comment by trulymobilenwp** [comment]
We will be approving the design and contributors to highlight in the carousel during today's marketing committee call. We previously agreed that the lines of code would not be a number highlighted on the page as @shabnammayel pointed out above. The final numbers used will come when the final specs are approved and the release is ready to go, but the three numbers will be on the page - projects, committers, and git repos.

**Comment by flora8984461** [comment]
Hello @karenmcnaughton and @shabnammayel ,
I made an initial animation on preview: https://deploy-preview-929--jakartaee.netlify.app/release/ 
Please let me know your thoughts on it, thanks!

**Comment by karenmcnaughton** [comment]
@chrisguindon @yi confirming that we want to remove  LINES OF CODE #, Tanja and I discussed. The animation looks good, we think the #9 can maybe be a little smaller.

**Comment by flora8984461** [comment]
@karenmcnaughton Thank you for your updates, I made the 9 smaller and removed the LINES OF CODE .
The preview will refresh soon.

**Comment by karenmcnaughton** [comment]
@flora8984461 as discussed earlier, if you can move the "Download Compatible Products" and "Specifications" up a little to make room for a text link below them: "See also: JakartaEE 8" [link to that page]

**Comment by flora8984461** [comment]
Hello Karen @karenmcnaughton  and Chris @chrisguindon ,
The site is updated as discussed. https://deploy-preview-929--jakartaee.netlify.app/release/9/ 
And I also hide committers section.

Thanks!
Yi

**Comment by flora8984461** [comment]
Hello Karen @karenmcnaughton and Chris @chrisguindon ,
Sorry I forgot to ask, what's the link to "watch the video interview"?

**Comment by chrisguindon** [comment]
@karenmcnaughton @TanjaObradovic the plan is to deploy https://deploy-preview-929--jakartaee.netlify.app/release/9/ on production today. This should allow the community to contribute the collaborator's section to the page.

Can you review the content and let us know if there is anything that we should change before that? I am thinking that we might want to add the release data on the page which we can remove on the release day.

The plan is to keep https://jakarta.ee/release/ as-is until the "official" launch for Jakarta 9.

> Hello Karen @karenmcnaughton and Chris @chrisguindon ,
> Sorry I forgot to ask, what's the link to "watch the video interview"?

Good question! @karenmcnaughton can you tell us if the current link is correct?

**Comment by TanjaObradovic** [comment]
@chrisguindon & @karenmcnaughton the necessary info is there, thanks. I am wandering if "See also: Jakarta EE 8" should be "Previous Releases: Jakarta EE 8"... It is not necessary to see that page and it is implied by "see also". Keep in mind this is very 
minor.

**Comment by flora8984461** [comment]
@TanjaObradovic  Thanks for your suggestion. I have updated the link text and the preview will refresh soon.

Please also help me check if the link of "watch the video interview" and "Download now" is correct, thx!

**Comment by karenmcnaughton** [comment]
@flora8984461 can we please remove the (top right orange) Download button on the page. It's confusing and takes them nowhere. Thanks! Refer to the landing page of the Jakarta EE 8 release--no Download button on the top right there.

**Comment by flora8984461** [comment]
> @flora8984461 can we please remove the (top right orange) Download button on the page. It's confusing and takes them nowhere. Thanks! Refer to the landing page of the Jakarta EE 8 release--no Download button on the top right there.

The change is updated and the [preview](https://deploy-preview-929--jakartaee.netlify.app/release/9/) is refreshed. Thanks for the feedback.
